UnitsNet/UnitDefinitions/SpecificEntropy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "Name": "SpecificEntropy",
+  "BaseUnit": "JoulePerKilogramKelvin",
+  "XmlDoc": "The SpecificEntropy",
+  "XmlDocRemarks": "Specific entropy is an amount of energy required to raise temperature of a substance by 1 Kelvin per unit mass",
+  "Units": [
+    {
+      "SingularName": "JoulePerKilogramKelvin",
+      "PluralName": "JoulesPerKilogramKelvin",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "J/(kg*K)" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "JoulePerKilogramDegreeCelsius",
+      "PluralName": "JoulesPerKilogramDegreeCelsius",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "J/(kg*C)" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "CaloriePerGramKelvin",
+      "PluralName": "CaloriesPerGramKelvin",
+      "FromUnitToBaseFunc": "x*4.184e3",
+      "FromBaseToUnitFunc": "x/4.184e3",
```

Comment:
I got ` 2.390057e-4` at: https://www.gordonengland.co.uk/conversion/specific_heat.htm
And `0.0002390057361377` at https://www.translatorscafe.com/unit-converter/en/specific-heat-capacity/1-6/joule%2Fkilogram%2FK-calorie%20(IT)%2Fgram%2F%C2%B0C/

Reply:
Yeah, it should be `"FromUnitToBaseFunc": "x*4.184e-3"`  instead of  `"FromUnitToBaseFunc": "x*4.184e3"`.

Reply:
But somehow I can't manage to make the tests pass when `x*4.184e3`. Hmm...

Reply:
Just asking to prevent any potential misunderstanding: Do you suggest to change the `2.39005e-4` with what you got, `2.390057e-4`, in `SpecificEntropyTests.cs`?

Reply:
I would use the must accurate value found in a references source ` 0.0002390057361377` in the test.
Then these JSON functions must also be updated as they seem wrong, but I don't know if they should be represented as some calculation or as the same constant.

Reply:
I couldn't spot the error in JSON function :/

Fixed values in tests.

Reply:
BTW, https://www.translatorscafe.com/unit-converter/en/specific-heat-capacity/1-6/joule%2Fkilogram%2FK-calorie%20(IT)%2Fgram%2F%C2%B0C/ gives `0.000238845896627` not `0.0002390057361377`.

I updated values in the test as `0.0002390057`.

I would like to leave this p.r. to you and @gratestas to finalize before merging, as I'm a bit confused.

Reply:
Careful, you linked to the IT calorie unit. It is *slightly* different than `Th` calories. I think we should add both and name them explicitly to avoid ambiguity.

Reply:
Oh, yeah, but I copied your link :)

I agree that we should fix this ambiguity in the entire library soon.

Reply:
Sorry, my bad. The JSON functions are probably good, at least they match the constant pretty well.